.travis.yml

```diff
@@ -68,13 +68,10 @@ addons:
 language: cpp
 compiler: clang
 
-before_install:
-    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
```

Comment:
agreed, was there some talk about a containerized image or something a
while ago?

On Fri, Sep 2, 2016 at 8:20 PM Michael Keller <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/betaflight/betaflight/pull/1111#discussion_r77430303>:
>
> > @@ -68,13 +68,10 @@ addons:
> >  language: cpp
> >  compiler: clang
> >
> > -before_install:
> > -    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
>
> Yep, true. Would be nice to be able to keep the toolchain around between
> builds, and get faster build times for it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/betaflight/betaflight/pull/1111/files/e98acee7cf4acea069c35485c3ce0212c7ac103d#r77430303>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAYZmi2_iFNeIlsjh4cESeUjAR7K03zoks5qmOeBgaJpZM4J0NUX>
> .
>